lib/DDG/Spice/Statista.pm

```diff
@@ -0,0 +1,26 @@
+package DDG::Spice::Statista;
+# ABSTRACT: Returns statistics from statista.com.
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "statistics about soccer";
+description "Shows example statistics";
+name "Statista";
+category "facts";
+
+my $limit = 16;
+my $lang = 2;
+
+triggers startend => "statistic", "statistics", "stats";
```

Comment:
To further explain I often check what queries exist in the autocomplete (a good determinant of popular queries) and all the autocomplete queries for "statistics" trigger the IA and show results, but they're almost entirely irrellevant in that the query isn't actually looking for stats articles:

![yolo__at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/8187102/17c6c28e-141c-11e5-8e69-df2562c0e090.png)